packages/cli/src/commonlib/log.ts

```diff
@@ -62,35 +70,35 @@ export class CLILogProvider implements LogProvider {
 
   white(msg: string): string {
     if (process.stdout.isTTY) {
```

Comment:
It seems `chalk` library has [auto-detection of ANSI escape sequence support](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#when_not_to_use_map), so this check is redundant now.